changelog/11603.trivial.rst

```diff
@@ -0,0 +1,7 @@
+Marked the ``TestLocalPath.test_make_numbered_dir_multiprocess_safe`` test as xfail.
```

Comment:
I don't think we need to announce this in the CHANGELOG at all, as this is internal to pytest and does not affect users. I will remove it.

Reply:
Done.

Reply:
Thanks! 🙏